### DISCUSSION TITLE
Importing beatmaps from the in-game browser

### DISCUSSION THREAD

**Discussion by VolcanicArts** [ppy/osu#16034] [Q&A] [answered]
When importing a beatmap from the in-game browser, if you're inside a collection it adds the beatmap to the open list of beatmaps, but doesn't add it to the collection.

Steps to recreate:
- Open a collection
- Download and open a beatmap from the in-game browser
- Right click the beatmap -> Collections -> It's not inside the collection, but still shows in the beatmap list

I'm not sure if this is intended, but it feels wrong, so I thought I'd report it.

**Comment by bdach** [comment]
not sure that if importing should associate a map with the current collection. the collection feels more like a filter to me.

**Reply by VolcanicArts** [reply]
I agree. I feel like it should deselect the collection you've opened when opening a beatmap from the in-game browser

**Reply by bdach** [reply]
well "presenting" a map from the beatmap listing will indeed bypass any currently active filters which i agree may be seen as confusing. not sure how to best address that.

**Comment by peppy** [comment, marked as answer]
This is correct behaviour. Clicking the notification to go to the beatmap will alreayd bypass the filter, too.

**Reply by VolcanicArts** [reply]
I see. Thank you!
